CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to the Lacework Terraform Modules
+
+### Table Of Contents
+
+* [Before getting started?](#before-getting-started)
+
+* [How to Contribute?](#how-can-i-contribute)
+    * [Reporting Bugs](#reporting-bugs)
+    * [Feature Requests](#feature-requests)
+    * [Pull Requests](#creating-a-pull-request)
+
+* [Developer Guidelines](/DEVELOPER_GUIDELINES.md)
+
+
+## Before getting started
+
+Read the [README.md](README.md) and
+ Hashicorps [best practices and syntax guidelines](https://www.terraform.io/docs/configuration/index.html)
+
+## Reporting Bugs
+
+Ensure the issue you are raising has not already been created under [issues](https://github.com/lacework/terraform-aws-config/issues).
+If no current issue addresses the problem, open a new [issue](https://github.com/lacework/terraform-aws-config/issues/new).
+Include as much relevant information as possible. See the [bug template](https://github.com/lacework/terraform-aws-config/blob/main/.github/ISSUE_TEMPLATE/bug_report.md) for help on creating a new issue.
+
+## Feature Requests
+
+If you wish to submit a request to add new functionality or an improvement to a terraform module then use the the [feature request](https://github.com/lacework/terraform-aws-config/blob/main/.github/ISSUE_TEMPLATE/feature_request.md) template to 
+open a new [issue](https://github.com/lacework/terraform-aws-config/issues/new)
+
+## Creating a Pull Request
+
+If you have made a change or added new functionality, you can submit pull request. The project maintainers will aim to review in a 2 week timeframe. When submitting a pull request please read the [developer guidelines](/DEVELOPER_GUIDELINES.md)
+The examples folder contains Terraform code that run as part of the CI pipeline. A new pull request will trigger this test run to ensure no breaking changes are added. We recommended sanity checking your own Terraform changes before submitting the change for review.
```

Comment:
Just noticed we need a carrier return to make a new paragraph 
```suggestion

The examples folder contains Terraform code that runs as part of the CI pipeline. A new pull request will trigger this test run to ensure no breaking changes are added. We recommended sanity checking your own Terraform changes before submitting the change for review.
```

Reply:
if not, it looks like this:
<img width="1018" alt="Screen Shot 2021-09-21 at 4 35 24 PM" src="https://user-images.githubusercontent.com/5712253/134191079-d083f04b-e118-4071-b19c-67bffaa459a1.png">